File: scripts/lib/analytics.js
// Script de Cloudflare Web Analytics, compartido por todos los generadores Node.
//
// Fuente única de verdad del snippet de analítica. Debe mantenerse idéntico a
// scripts/lib/analytics.py (versión Python) y al que lleva ya insertado el
// HTML ya publicado (ver scripts/migrate-analytics-2026-08.js, que lo inserta
// retroactivamente en todo el HTML ya generado). Token obtenido en Cloudflare
// > Analytics & Logs > Web Analytics > alhaurinaldia.es (modo manual: el
// automático no llegaba a inyectarse vía GitHub Pages).

const CF_ANALYTICS_SNIPPET = `<script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "dc9eeda336f943ca87175a3b83faee35"}'></script>`;

module.exports = { CF_ANALYTICS_SNIPPET };

File: scripts/lib/analytics.py
"""Script de Cloudflare Web Analytics, compartido por todos los generadores Python.

Fuente unica de verdad del snippet de analitica. Debe mantenerse identico a
scripts/lib/analytics.js (version Node) y al que llevan ya insertados las
paginas generadas a mano (ver scripts/migrate-analytics-2026-08.py, que lo
inserto retroactivamente en todo el HTML ya publicado). Token obtenido en
Cloudflare > Analytics & Logs > Web Analytics > alhaurinaldia.es (modo manual,
el automatico no llegaba a inyectarse via GitHub Pages).
"""

CF_ANALYTICS_SNIPPET = (
    "<script defer src='https://static.cloudflareinsights.com/beacon.min.js' "
    'data-cf-beacon=\'{"token": "dc9eeda336f943ca87175a3b83faee35"}\'></script>'
)
